### DISCUSSION TITLE
About ScalesGrid of Chart

### DISCUSSION THREAD

**Discussion by dlyltm** [dotnetcore/BootstrapBlazor#2257] [Q&A] [answered]
How to make the XScalesGrid and YScalesGrid of the Chart disappear completely?

**Comment by ArgoZhang** [comment]
@dlyltm try
```
var ds = new ChartDataSource();
ds.Options.ShowXScales = false;
ds.Options.ShowYScales = false;
```

**Reply by dlyltm** [reply]
I just want the grid lines outside the coordinate axis not to be displayed, but the coordinate axis itself to be displayed. Doing this doesn't even show the coordinate axis.

**Comment by ArgoZhang** [comment, marked as answer]
@dlyltm 

like this?

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/a5cbc3b9-32a4-4ad6-9aff-dc30c9338e76)

```
ShowX/YLine = false
```

**Reply by dlyltm** [reply]
Yes，it is.
